game/games/testgame/balloonPop.gd

```diff
@@ -0,0 +1,115 @@
+# warning-ignore-all:return_value_discarded
+extends MarginContainer
+
+const END_MESSAGE := "You got %s points!"
+const STATUS_MESSAGE := "You'r in stage %s/10 and have %s points!"
+
+var balloon_scene := preload("res://games/testgame/balloon.tscn")
+
+var colors := {
+	"Red": Color.red,
+	"Green": Color.green,
+	"Yellow": Color.yellow,
+	"Blue": Color.blue,
+	"Purple": Color.purple,
+	"Orange": Color.orange,
+}
+
+var points := 0
+var stage := 0
+var search_color: Color
+
+onready var _timer := $Timer
+onready var _respawn_timer := $respawnTimer
```

Comment:
```suggestion
onready var _respawn_timer := $RespawnTimer
```
typo probably while linting. Starting with lower case popping a balloon will result in an error because of a null instance.